Documentation/contributing/release/backports.rst

```diff
@@ -76,10 +76,24 @@ process for backporting these PRs:
 One-time Setup
 ~~~~~~~~~~~~~~
 
+#. Make sure you have a GitHub developer access token with the ``public_repos``
+   ``workflow`` scopes available. You can do this directly from
+   https://github.com/settings/tokens or by opening GitHub and then navigating
+   to: User Profile -> Settings -> Developer Settings -> Personal access token
+   -> Generate new token.
+
 #. The scripts referred to below need to be run on Linux, they do not work on
-   macOS. It is recommended to use the dev VM (``contrib/vagrant/start.sh``),
-   as it will have all the correct versions of dependencies / libraries. Once
-   you have a machine ready, you need to configure git to have your name and
+   macOS. It is recommended to create a container using (``contrib/backporting/Dockerfile``),
+   as it will have all the correct versions of dependencies / libraries. In order to
+   create a container based setup, build and run a container:
+
+   .. code-block:: shell-session
+
+      $ docker build --build-arg SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" -t cilium-backport contrib/backporting/.
```

Comment:
Could we not mount the private key inside the container instead of doing this? This approach will copy the private key in various places in the filesystem, I'm not very comfortable with that :grimacing: It will for instance be visible in `docker history cilium-backport`.

Reply:
> I think we need to change the way we pass the private key.

> Could we not mount the private key inside the container instead of doing this?

Yeah, I was thinking about this. I was originally thinking of COPYing users's private key into the image, but Docker only allows COPYing files from the directory you are building an image from. I was a bit hesitant to mount user's entire ~/.ssh folder, but could be an option. I'll explore this a bit. Thanks!